claragenomics/dl4atac/train/metrics.py

```diff
@@ -255,9 +257,14 @@ def __init__(self):
     def __call__(self, x, y):
         x = Metric.convert_to_numpy(x)
         y = Metric.convert_to_numpy(y)
-        x = x.flatten()
-        y = y.flatten()
-        self.val = scipy.stats.spearmanr(y, x)[0]
+        x = np.apply_along_axis(rankdata,0,x)
+        y = np.apply_along_axis(rankdata,0,y)
+        xm = x.mean()
+        ym = y.mean()
+        np.subtract(x, xm, out=x)
```

Comment:
is this the in-place calculation that caused the issue with backprop during training?

Reply:
No, the problem was in `CorrCoef` which is now changed back.